pkg/s3select/helpers.go

```diff
@@ -70,42 +72,28 @@ func representsInt(s string) bool {
 	return false
 }
 
-// The function below processes the where clause into an acutal boolean given a
-// row
-func matchesMyWhereClause(row []string, columnNames map[string]int, alias string, whereClause interface{}) (bool, error) {
-	// This particular logic deals with the details of casting, e.g if we have to
-	// cast a column of string numbers into int's for comparison.
+// matchesMyWhereClause takes map[string]interfaces{} , process the where clause and returns true if the row suffices
+func matchesMyWhereClause(individualRecord map[string]interface{}, alias string, whereClause interface{}) (bool, error) {
 	var conversionColumn string
-	var operator string
+	var operator string //It gives the right value of where clause
```

Comment:
Comments should be `//<space>` native Go style

Reply:
Corrected.